### PR TITLE
badge component

### PR DESCRIPTION
**Pull request by devAmlan**
This is a badge component. Please review my PR.

### CODE REVIEW

File: library/component/badge/badge.css [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+      border-radius:50%;
+      text-align:center;
+      top:0;
+      left:0;
+      z-index:2;
+  }
+  .right_badge_text{
+    position:absolute; 
+    background-color:var(--danger-color);
+    color:white;
```
Comment (itsmebipul00) [inline library/component/badge/badge.css:23]: var to be used here

Comment (devAmlan) [reply library/component/badge/badge.css:23]: yeah I will do it, thanks bipul

**Review by Muzammil-khan710 — APPROVED** [review]
Good to merge!

File: library/component/badge/badge.css [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+    border-radius:50%;
+  }
+  .badge_avatar_successdot{
+    width:1rem;
+    height:1rem;
+    background-color:var(--success-color);
+    position:absolute;
+    bottom:0%;
+    right:15%;
+    border:1px solid var(--light);
```
Comment (zoef-yt) [inline library/component/badge/badge.css:50]: var(--light) make less sense maybe add colour at the start or end to show that it's a colour.

**Review by zoef-yt — CHANGES_REQUESTED** [review]

File: library/component/badge/badge.css (hunk 2 above)
Comment (devAmlan) [reply library/component/badge/badge.css:50]: ok I will do it , thanks zoef.

**Pull request merged**